app/controllers/accept_account_invitations_controller.rb

```diff
@@ -16,6 +23,6 @@ def update
   private
 
   helper_method memoize def received_invitation
-    AccountInvitation.unaccepted.for(current_user).find_by!(token: ps.fetch(:token))
+    AccountInvitation.unaccepted.for(current_user).find_by!(token: ps[:token])
```

Comment:
what was the problem with the previous implementation? revert it?

Reply:
... ok, I've seen the main idea of the`fetch` for us: errors. Ok